libs/challenge-registry/user-profile/src/lib/user-profile.component.scss

```diff
@@ -198,8 +198,9 @@ $dl-radius-radius-round: 50%;
 .base-main-section {
   top: 729px;
   left: 495px;
-  width: 100%;
+  width: calc(100% - 495px);
```

Comment:
Could `495px` be replaced by a variable? If it is meant to be equal to `left`, the layout could easily break if one of the two values is changed but not the other.

Reply:
hmm, it's actually an easy way to prevent from breaking the layout. If we stretch width, the right content take the rest of space. If we use variable, that means the width will be fixed on the right content. 

Also, our entire page is using the arbitrary px and absolute position from teleportHQ. I don't think we would change just couple of values, which will probably break the layout anyways :).

Reply:
I know you know that duplicated values increase the risk of breaking code. Maybe the way you generate/integrate the code reduces this risk. I leave it to up to you.